packages/client/src/components/dialogs/SalesDepartmentRequestDialog/index.js

```diff
@@ -110,6 +110,7 @@ const SalesDepartmentRequestDialog = ({
           errorMessage={t("Common:RequiredField")}
         >
           <TextInput
+            id="name"
```

Comment:
Too simple id value (potential duplicates)

Reply:
Fix to "your-name"